notebooks/notebooks/iclr2022/estimate-flops-adv.ipynb

```diff
@@ -0,0 +1,336 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.        adv_net_size: str = "b6c96" if info["adv_name"] != "random" else "random"
</pre><p>Do we never use larger networks? I thought we at least tested b10 -- did we never use them in any major training runs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>We have tested b10 in a couple runs, and even larger networks in Tom's warmstart runs. But all of the runs currently in our paper use b6 as the adversary.</p><p>Unfortunately we don't track the adversary net size in our metadata, so we have to specify what adversary net size is used to compute stats. I made the adv_net_size a variable to be passed in instead of hardcoded.</p>

Reply:
<p>Making it a parameter sounds good. Might be worth adding to our metadata for future runs, though probably little benefit at this point as we'll still need to handle the old runs.</p>